src/Implementations/MethodDescriptor.php

```diff
@@ -23,6 +23,11 @@ class MethodDescriptor implements MDI
      */
     private $return;
 
+    /**
+     * @var boolean
+     */
+    private $static;
+
     /**
      * @var Type[]
```

Comment:
ClassType

Reply:
there is no reason why there shouldn't be none-classparameters for example